swagger.yaml

```diff
@@ -0,0 +1,154 @@
+swagger: "2.0"
+info:
+  description: "Authors Haven - A social platform for the creative at heart."
+  version: "1.0.0"
+  title: "Authors Haven"
+  license:
+    name: "MIT"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+host: "kingsmen-ah-backend-staging.herokuapp.com"
```

Comment:
I remember commenting that you should remove this

Reply:
Hello, @fidelisojeah I removed it. Remember you flagged it on the swagger.json configuration file. this file is the swagger.yaml file and it returns structural error if it is do like so

<img width="719" alt="Screenshot 2019-06-06 at 11 55 14 AM" src="https://user-images.githubusercontent.com/22410701/59027888-19058a00-8852-11e9-9300-b6b1a8516ce0.png">

Reply:
You understand what `remove` means yes? and you can see the file I am commenting on?